src/Nri/Ui/Heading/V1.elm

```diff
@@ -0,0 +1,190 @@
+module Nri.Ui.Heading.V1 exposing
+    ( Heading, heading
+    , withVisualLevel, VisualLevel(..)
+    , withDocumentLevel, DocumentLevel(..)
+    , view
+    )
+
+{-| Headings such as you'd find in Nri.Ui.Text.V3, but customization options for
+accessibility.
+
+@docs Heading, heading
+
+@docs withVisualLevel, VisualLevel
+
+@docs withDocumentLevel, DocumentLevel
+
+@docs view
+
+-}
+
+import Css exposing (..)
+import Html.Styled exposing (..)
+import Html.Styled.Attributes exposing (css)
+import Nri.Ui.Colors.V1 exposing (..)
+import Nri.Ui.Fonts.V1 as Fonts
+
+
+{-| -}
+type Heading msg
+    = Heading (List (Html msg)) VisualLevel DocumentLevel
+
+
+{-| start a heading. Render it with `view`.
+-}
+heading : List (Html msg) -> Heading msg
+heading content =
+    Heading content Top H1
+
+
+
+-- VISUAL LEVEL
+
+
+{-| -}
+type VisualLevel
```

Comment:
I'm curious - where did these names come from? I already forget the visual hierarchy when trying to add headings to my pages

Reply:
they came from the `Nri.Ui.Text.V3` names. They'll be listed in order in the docs!